en/datatypes/float.adoc

```diff
@@ -54,12 +58,12 @@ Single quote `'` is used as a place value seperator for long integers, and can a
 
 == Comparisons
 
-All comparators can be applied on floats: `=, ==, <>, >, <, >=, &lt;=, =?`. In addition, `min`, and `max` are also supported.
+All comparators can be applied on `float!`: `=, ==, <>, >, <, >=, &lt;=, =?`. In addition, `min`, and `max` are also supported.
```

Comment:
Let's think about "applied on" vs "applied to", or even "Floats support all comparators". More active voice than "can be...".

Reply:
I'll think on that